### DISCUSSION TITLE
docker-compose configuration file and docker image for Chyrp Lite

### DISCUSSION THREAD

**Discussion by denisebitca** [xenocrat/chyrp-lite#177] [Show and tell]
*(EDIT: I am far from being the first person to do this - but for now my version is more up to date)*

Hello,

I'm in the process of setting up a blog for myself, and since my server's technical stack uses Docker containers, I made a docker-compose file to allow for running a containerised version of the latest release of Chyrp Lite: [docker-chyrp-lite](https://github.com/denisebitca/docker-chyrp-lite).

I'm sharing this in hopes that this will be useful for more people - if you find it useful, don't hesitate to contribute more ideas to the file :)

Thank you for reading!

**Comment by xenocrat** [comment]
Thank you for sharing!
